spec/requests/posts_request_spec.rb

```diff
@@ -1,17 +1,47 @@
 require 'rails_helper'
 
 RSpec.describe 'Posts', type: :request do
-  describe 'GET /index' do
+  subject do
+    User.new(name: 'Esi Ogagaoghene', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Nigeria.',
+             posts_counters: 0)
+  end
+
```

Comment:
- [ ] As you did for the users_spec kindly make sure your posts Spec file includes tests that ensure the correct view template is rendered. Example : `expect(response).to render_template(:show)`

Reply:
@BenMukebo Just be creative I decided not to follow same rule hence I used another approach. But since you are insisting, I will approach it as said.